### ISSUE TITLE
The bounds of the  layer that are sampled with the sampler never include the FX of that layer

### ISSUE THREAD

**Issue by pineapplespatula** [adobe-photoshop/spaces-design#3109]
If you want to copy a drop shadow, it would seem like you should be able to click on the drop shadow. 

instead we get this
![policybounds](https://cloud.githubusercontent.com/assets/1008531/10704971/5f5e43b4-798f-11e5-82a3-cd8fcdbd4ee1.gif)

**Comment by volfied** [comment]
Looking into this, it depends on whether hittest returns anything to us.

**Comment by volfied** [comment]
Yeah, PS hitTest APIs don't return the ID of a layer when hovering over it's shadow. There isn't anything we can do on our side.

**Comment by chadrolfs** [comment]
so do we want to log a watson bug and try to fix this or close this as designed?

**Comment by pineapplespatula** [comment]
I would say we add a trello card or a watson bug to the core list, and see what is possible on the core side before closing this one out
